src/domain_classes/dimension.py

```diff
@@ -10,8 +10,8 @@ def _get_data_type_from_dmt_type(
         return type_enum.to_py_type()
     except ValueError:
         return attribute_type
-    except Exception as error:
-        raise Exception(f"Something went wrong trying to fetch data type: {error}")
+    except Exception as ex:
```

Comment:
should we not avoid catching Exception, as it is way too general, and will catch even "ctrl+c" force quit etc. 

Also I think the exception message "data type: {ex}" does not quite make sense.

Reply:
Agreed, but don't want to touch logic with this PR :)